lib/benchmark/ips.rb

```diff
@@ -73,6 +76,34 @@ def ips(*args)
       report
     end
 
+    # Quickly compare multiple methods on the same object.
+    # @param methods [Array<Symbol>] An array of method names (as symbols) to compare.
+    # @param receiver [Object] The object on which to call the methods. Defaults to Kernel.
+    # @param opts [Hash] Additional options for customizing the benchmark.
+    # @option opts [Integer] :warmup The number of seconds to warm up the benchmark. (optional)
+    # @option opts [Integer] :time The number of seconds to run the benchmark. (optional)
```

Comment:
IMO these are really expressed so much more simplify without a kwargs hash.

This is so much more clear, the defaults are explicitly shown, and it doesn't need yarddoc to interpret:

```suggestion
    def ips_quick(methods, receiver = Kernel, warmup: 2, time: 5)
```